lib/katello_deploy/repositories.rb

```diff
@@ -22,8 +23,8 @@ def configure(koji_repos = false)
       if koji_repos
         setup_koji_repos(@os_version, @katello_version, foreman_version)
       else
-        bootstrap_katello(@katello_version, @os_version)
-        bootstrap_foreman(foreman_version, @os_version)
+        bootstrap_katello(@katello_version, @os_version) if @katello_version
+        bootstrap_foreman(@foreman_version, @os_version) if @foreman_version
```

Comment:
I updated the logic, incorporating some of the thought from your other PR, to better handle the Foreman repo setup based on version. Take a look and let me know what you think.